src/Avalonia.Visuals/Point.cs

```diff
@@ -269,18 +269,18 @@ public Point WithY(double y)
         }
 
         /// <summary>
-        /// Deconstructor for decomposition Point
+        /// Deconstructs the point into it's X and Y coordinates.
```

Comment:
Nit: should be "its" (similarly with others)

Reply:
Other than that, it all looks good to me.

Reply:
Thank you, already fixed